core/runtime/dynamic_map_hash.odin

```diff
@@ -0,0 +1,147 @@
+package runtime
+
+import "core:intrinsics"
+import "core:math/bits"
```

Comment:
Please do not important anything into `core:runtime` except for `core:intrinsics`. Duplicate the code if necessary.